performance-evaluation/constants.py

```diff
@@ -34,45 +35,83 @@
 OPERATION_COPY_ANSWERS = "OPERATION_COPY_ANSWERS"
 OPERATION_COPY_FEEDBACK = "OPERATION_COPY_FEEDBACK"
 
+AUTO_EVALUATION_FOLDER_KEY = "126gcCwdBklMyEnIgdYWSPEiwSD_jy-Z7"
+MANAGER_EVALUATION_FOLDER_KEY = "1posz9Fkla1VZexbwQP32q0xo5aaDBSox"
+EXCHANGE_EVALUATION_FOLDER_KEY = "1nLjfe7kcnsRHRof2kFz06DAmcMzK2Kfr"
+RID_EVALUATION_FOLDER_KEY = "1hHBXRBU2ls5CVcHHfN4lzlNh6S20JeIS"
+PERFORMANCE_STUDY_FOLDER_KEY = "1wGrZ9n-il5YnaS6Z_ihapCg1z81ad5Bn"
+PERFORMANCE_REPORT_FOLDER_KEY = "0B-gOApkHyMPvelYzM3VGV2ZlYVk"
 
 operations_by_mode_dictionary = {
-	OPERATION_COPY_TABS: 				[NEXT_EVALUATION, AUTO_EVALUATION, MANAGER_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION, RID_EVALUATION],
-	OPERATION_BUILD_EVALUATION_FORM:	[EXCHANGE_EVALUATION, FIRST_EVALUATION],
-	OPERATION_HIDE_TALENTS: 			[NEXT_EVALUATION, AUTO_EVALUATION, MANAGER_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION],
-	OPERATION_COPY_ANSWERS: 			[NEXT_EVALUATION, AUTO_EVALUATION, MANAGER_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION],
-	OPERATION_COPY_FEEDBACK:			[NEXT_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION, UPDATE_FEEDBACK],
+    OPERATION_COPY_TABS: 				[NEXT_EVALUATION, AUTO_EVALUATION, MANAGER_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION, RID_EVALUATION],
+    OPERATION_BUILD_EVALUATION_FORM:	[EXCHANGE_EVALUATION, FIRST_EVALUATION],
+    OPERATION_HIDE_TALENTS: 			[NEXT_EVALUATION, AUTO_EVALUATION, MANAGER_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION],
+    OPERATION_COPY_ANSWERS: 			[NEXT_EVALUATION, AUTO_EVALUATION, MANAGER_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION],
+    OPERATION_COPY_FEEDBACK:			[NEXT_EVALUATION, EXCHANGE_EVALUATION, FIRST_EVALUATION, UPDATE_FEEDBACK]
 }
 
+folders_dictionary = {
+    NEXT_EVALUATION: {
+		'need_child_key': True,
+        'parent_key': False,
```

Comment:
There's a whitespace error in this keys, I'm guessing tabs and spaces mixed up. The entire `folders_dictionary` has some weird spacing errors.

Reply:
This i have no f** idea why is happening. I fixed but still is showing wrong